### DISCUSSION TITLE
Teams is in examples but... webcam?

### DISCUSSION THREAD

**Discussion by stefer09** [microsoft/wslg#526] [Q&A]
I can't pick a webcam in Teams, is there a way to pass it to Ubuntu in WSLg ?

**Comment by spronovo** [comment]
Hi @stefer09,

We do not currently support sharing a USB camera between the Windows host and WSL. However, a few days ago we announced support for USB device assignment in WSL. Please see this [blog post](https://devblogs.microsoft.com/commandline/connecting-usb-devices-to-wsl/) if you missed it. You could use USB device assignment to assign a USB camera to WSL... keeping in mind that this assignment is exclusive, which means the Windows host will not be able to use the USB camera while it is assigned to WSL. The blog has instructions on how to get started assigning USB devices.
